delete-rgs-query.azcli

```diff
@@ -0,0 +1,22 @@
+# Mega powers to delete across subscriptions and e.g. groups without tags - USE WITH CAUTION
+# Requires the Resource Graph extension (az extension add --name resource-graph)
+# More info: https://docs.microsoft.com/en-us/azure/governance/resource-graph/first-query-azurecli
+
+# This command allows you to delete resource groups, given any query against the Azure Resource Graph.
+# Step to run the command:
+# 1. Make sure to input the subscriptionId you want to target `subscriptionId=<subscriptionId>`
+# 2. Change the query to find the resource groups to delete e.g. `where tags !contains 'protect'`
+# 3. Run the command. You will be asked to confirm each deletion.
+
+# Description of the command:
+# az graph query -q "ResourceContainers | where type == 'microsoft.resources/subscriptions/resourcegroups'| where subscriptionId == '<subscriptionId to target>' \ # Query for RGs in a specific subscription
+#  | where tags !contains 'protect' \ # The condition for which RGs to return
```

Comment:
Can you change this to `where tags contains 'delete'`? To be consistent with the other version of this and to be less risky.  So user has to explicitly add a tag to delete versus not a tag to delete.

No worries if you are maxed out on revs for this. I can take it over :)

Reply:
Done